### PR TITLE
feat: adding imported tab in models page

### PR DESCRIPTION
**Pull request by axel7083**
### What does this PR do?

Adding an imported tab to models page for imported models.

### Screenshot / video of UI

https://github.com/containers/podman-desktop-extension-ai-lab/assets/42176370/f494ff6c-fbc9-4f18-9d2c-127cd883c5eb

### What issues does this PR fix or reference?

Fixes https://github.com/containers/podman-desktop-extension-ai-lab/issues/944

### How to test this PR?

- [x] unit tests cover this feature

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
Works like a charm, great work!

File: packages/frontend/src/pages/Models.svelte [hunk 1]
```diff
@@ -145,6 +147,15 @@ async function importModel() {
               {/if}
             </Route>
 
+            <!-- Imported models -->
+            <Route path="/imported">
+              {#if importedModels.length > 0}
+                <Table kind="model" data="{importedModels}" columns="{columns}" row="{row}"></Table>
+              {:else}
+                <div role="status">There is no model yet</div>
```
Comment (jeffmaury) [inline packages/frontend/src/pages/Models.svelte:155]: @cdrage suggests to use There are no models yet

Comment (axel7083) [reply packages/frontend/src/pages/Models.svelte:155]: This is a copy paste of the other tab, another PR would be required for the others tab as this is not the scope

Comment (axel7083) [inline packages/frontend/src/pages/Models.svelte:155]:
```suggestion
                <div role="status">There are no models yet</div>
```

File: packages/frontend/src/pages/Models.svelte [hunk 2]
```diff
@@ -145,6 +147,15 @@ async function importModel() {
               {/if}
             </Route>
 
+            <!-- Imported models -->
+            <Route path="/imported">
+              {#if importedModels.length > 0}
+                <Table kind="model" data="{importedModels}" columns="{columns}" row="{row}"></Table>
+              {:else}
+                <div role="status">There are no models yet</div>
```
Comment (feloy) [inline packages/frontend/src/pages/Models.svelte:155]: I'm not sure which expression is correct, but this one is different from the one in other tabs ('There is no model yet')

**Review by feloy — APPROVED** [review]

**Pull request merged**